packages/block-renderer/src/hooks/use-rendered-patterns.ts

```diff
@@ -41,7 +42,7 @@ const useRenderedPatterns = (
 	const queries = Object.entries( patternIdsByCategory ).map( ( [ category, patternIds ] ) => ( {
 		queryKey: [ 'rendered-patterns', siteId, stylesheet, category, patternIds, siteInfo ],
 		queryFn: () => fetchRenderedPatterns( siteId, stylesheet, category, patternIds, siteInfo ),
-		staleTime: 1000 * 60 * 5, // 5 minutes
+		staleTime: isEnabled( 'pattern-assembler/skip-rendered-pattern-cache' ) ? 0 : 1000 * 60 * 5, // 5 minutes
```

Comment:
[nit] We can simply call it `pattern-assembler/disable-cache` and then we can apply the same flag to other places if needed